keyboards/preonic/keymaps/solstice22/config.h

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
This whole file should be deleted, it's not overriding anything from the keyboard-level config.h.